packit_service/service/events.py

```diff
@@ -240,7 +241,7 @@ def __init__(
         self.target_repo = target_repo
         self.commit_sha = commit_sha
         self.github_login = github_login
-        self.ref = pr_id
+        self.ref = None
```

Comment:
Does this mean that for *PullRequestEvent*s there is no `ref`? Is `ref` a Git-ref?

I'm trying to understand why "Use None as a ref for PR events" is required. Maybe it would be nice to provide more context on this change in the commit message?

Reply:
- Yes, I wanted to rename that to `git_ref` -- It looks like it would be useful.
- With git-refs for PRs, it's harder. (We are using that ref for checkout and PR needs to be checkout-ed differently.)

The problem was here, that we first checkout `ref` and then the PR:
https://github.com/packit-service/packit/blob/ee02ce960cb6ef42b6cdf30504ac1e7ff2edb56e/packit/local_project.py#L123-L126

Reply:
I understand now. Thanks for the explanation!

Reply:
I've introduced the two properties:
- the one for git checkout
- the second one for identification and presentation purpose (e.g. copr project name, logs, ...)